src/App.js

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import Header from './components/Header';
+import Cards from './components/Cards';
+import Footer from './components/Footer';
+import Instruction from './components/Instruction';
+
+function App() {
+  const [cards, setCards] = useState([]);
+
+  //Delete Card
+  const deleteCard = (card) => {
+    var array = [...cards];
+    var index = array.indexOf(card);
+    if (index !== -1) {
+      array.splice(index, 1);
+      setCards(array);
+    }
+  };
+
+  //Add Card
+  const addCard = () => {
+    const newArr = [...cards];
+    newArr.push({
+      number: Math.floor(Math.random() * 100),
+    });
+    setCards(newArr);
+  };
+
+  //Sort Cards
+  const sortCard = () => {
+    const sorted = [...cards].sort((a, b) => {
+      return b.number - a.number;
```

Comment:
you can write this shorter. When using arrow functions, just remove curly braces as well as the return statement.